### PR TITLE
Recipe on how to generate InChIKeys from IUPAC names

### PR DESCRIPTION
**Pull request by egonw**
Implements https://github.com/FAIRplus/the-fair-cookbook/issues/425

### CODE REVIEW

**Comment by egonw** [comment]
Rebased on `develop`.

**Review by terazus — APPROVED** [review]

File: content/recipes/infrastructure/iupac-names.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+(fcb-infra-iupac)=
+# Generating InChIKeys for IUPAC names
+
++++
+<br/>
+
+````{panels_fairplus}
+:identifier_text: FCBxxx
```
Comment (proccaserra) [inline content/recipes/infrastructure/iupac-names.md:8]:
```suggestion
:identifier_text: FCB080
```

File: content/recipes/infrastructure/iupac-names.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+(fcb-infra-iupac)=
+# Generating InChIKeys for IUPAC names
+
++++
+<br/>
+
+````{panels_fairplus}
+:identifier_text: FCB080
+:identifier_link: 'https://w3id.org/faircookbook/FCBxxx'
```
Comment (proccaserra) [inline content/recipes/infrastructure/iupac-names.md:9]:
```suggestion
:identifier_link: 'https://w3id.org/faircookbook/FCB080'
```

File: _toc.yml [hunk 3]
```diff
@@ -38,6 +38,8 @@
         title: Unique, persistent identifiers #FCB006
       - file: content/recipes/infrastructure/chemical-identities
         title: InChI and SMILES identifiers for chemical structures #FCB007
+      - file: content/recipes/infrastructure/iupac-names
+        title: Generating InChIKeys for IUPAC names #FCBxxx
```
Comment (proccaserra) [inline _toc.yml:42]:
```suggestion
        title: Generating InChIKeys for IUPAC names #FCB080
```

**Review by proccaserra — APPROVED** [review]

**Pull request merged**